src/components/icons/pin-arrow.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export const paths = {
+  // up arrow icon
+  upArrow:
+    'm10 4.167 4.583 4.583-1.146 1.146-2.604-2.604v9.375H9.167V7.292L6.563 9.896 5.417 8.75z',
+  // down arrow icon
```

Comment:
Think you can remove these comments, as the object property names are self explanatory.